public/mix-manifest.json

```diff
@@ -1,7 +1,10 @@
 {
     "/js/app.js": "/js/app.js",
     "/css/app.css": "/css/app.css",
+    "/images/download-stats-black.svg": "/images/download-stats-black.svg",
+    "/images/download-stats-white.svg": "/images/download-stats-white.svg",
     "/images/download-stats.svg": "/images/download-stats.svg",
     "/images/mismatch-finder-logo.svg": "/images/mismatch-finder-logo.svg",
-    "/images/user.svg": "/images/user.svg"
+    "/images/user.svg": "/images/user.svg",
+    "/images/wikidata-logo.svg": "/images/wikidata-logo.svg"
```

Comment:
suggestion: This line about the logo seems unrelated to this change.

I assume it was added by mistake and should probably be dropped?

Reply:
oh, yes, i don't know where this is coming from, since this file is automatically generated.

Reply:
removed!